BackendApartmentReservation/BackendApartmentReservation/Managers/ChecklistManager.cs

```diff
@@ -59,7 +64,30 @@ public class ChecklistManager : IChecklistManager
                 _logger.LogInformation($"Created car rent amenity for employee {employee.Id}.");
             }
 
-            await _cheklistRepository.AddChecklist(checklist);
+            if (livingPlaceReservationInfo.Required) { 

```

Comment:
Hmm. Okay, I interpreted things differently. We have a requirement, that apartments have a priority over hotel. So I thought maybe we will be trying to get an apartment first, then if no places left, go for hotel?

Reply:
Good catch. There is also a variant that LivingPlace is not needet at all, same as with Cars and Flights. Maybe just leave it for Admin to decide if Apartment or Hotel is needed when Reservations are sent? Later we will be adding some kind of timetable with Free/Booked timelines for Apartments, maybe then this can be adjusted as well

Reply:
Possible. But I dont know which option is easier at this stage

Reply:
Well I thought about that, looked at our backlog, and yes, I agree with you. Let us leave it so that admin decides which living place to book. Later on, we will add reservations, then someone will have to refactor code.